src/conventions.jl

```diff
@@ -0,0 +1,33 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+Convention used to reconstruct the faces of a polytope
+from a list of vertices.
+"""
+abstract type OrderingConvention end
+
+"""
+    connectivities(polytope, convention, k)
+
+Return the list of `Connectivity` elements used to construct all
+faces of rank `rank` of a polytope, following an ordering convention.
+"""
+function connectivities(polytope::Type{<:Polytope}, rank::Integer, convention::OrderingConvention)
```

Comment:
What about breaking a line before the last argument to emphasize the convention?